pkg/cloudprovider/providers/azure/azure.go

```diff
@@ -47,14 +47,15 @@ import (
 
 const (
 	// CloudProviderName is the value used for the --cloud-provider flag
-	CloudProviderName            = "azure"
-	rateLimitQPSDefault          = 1.0
-	rateLimitBucketDefault       = 5
-	backoffRetriesDefault        = 6
-	backoffExponentDefault       = 1.5
-	backoffDurationDefault       = 5 // in seconds
-	backoffJitterDefault         = 1.0
-	maximumLoadBalancerRuleCount = 148 // According to Azure LB rule default limit
+	CloudProviderName      = "azure"
+	rateLimitQPSDefault    = 1.0
+	rateLimitBucketDefault = 5
+	backoffRetriesDefault  = 6
+	backoffExponentDefault = 1.5
+	backoffDurationDefault = 5 // in seconds
+	backoffJitterDefault   = 1.0
+	// According to https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits#load-balancer.
+	maximumLoadBalancerRuleCount = 250
```

Comment:
Is this one to one mapping with a K8s service of type LoadBalancer ?

Reply:
one to one, or one to many. e.g. if service tags or loadBalancerSourceRanges are set, multiple rules would be created for the service

Reply:
Hi Feiskyer,

My name's Vinicius Silva from Brazil and I'm following your commits on Kubernetes Respository, thank your for works to make better every day the Kubernetes. Today our company is a  Microsoft partner and we are heaving user of AKS service. Yesterday we found a limit of LoadBalancer to use in Aks (Actual number 250) and we have a little question about.

You can help us to understand these limit? Follow the little code part about the limits of loadbalancers:

File: azure.go
line: 50:                    maximumLoadBalancerRuleCount = 250
line: 405
if az.MaximumLoadBalancerRuleCount == 0 {
az.MaximumLoadBalancerRuleCount = maximumLoadBalancerRuleCount
}

It's possible we use this value on flag for example? Other question is, the LoadBalancer basic are documented on Microsoft docs the limit is 250, if we use Standard version, we can use the 1500 (Rules per resouce) for example?

Image 1: Load Balancer Documentation (https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits#load-balancer)

Reply:
yep, it could be configured in azure cloud configure file. just set `maximumLoadBalancerRuleCount` to the new value there.